documentation/deliberately_vulnerable_flask_app/setup.sh

```diff
@@ -7,11 +7,10 @@
 # Source this file to run it:
 # . ./setup.sh
 
-# This file should ONLY include standard setup steps which would be used by a
-# real Flask project. Do not add anything Pysa/Pyre specific.
-
-python3 -m venv venv
-# shellcheck disable=SC1091
-. venv/bin/activate
-pip install Flask
+pip install -r requirements.txt
 rm ../../.pyre_configuration
+# Create .pyre_configuration with as ../../stubs as taint_models_path
+echo '{"source_directories": ["."], "taint_models_path": "../../stubs"}' > ./.pyre_configuration
+cp ../../tools/integration_tests/run_integration_tests.py ./
```

Comment:
Why are we copying the file here? Can't we directly run the python script from `tools/integration_tests/run_integration_tests.py` and set the `source_directories` in the pyre config file to point to `documentation/deliberately_vulnerable_flask_app`?

Reply:
I tried staying at `documentation/deliberately_vulnerable_flask_app` and running `python3 ../../run_integration_tests.py` but it gave me strange errors such as `taint.config` file not being found etc.

Reply:
I would like to avoid having to copy the integration test file to make it work because I can imagine a scenario in which we want to run it manually and it's a bit inconvenient to have to copy it in a different folder to make it work.
I think the issue is that pyre checks the `.pyre_configuration` file inside `tools/integration_tests/` and is not able to find it becuase our  `.pyre_configuration` is in `documentation/deliberately_vulnerable_flask_app`
I can see two possible approaches (not sure which one work better):
1. Have the `run_integration_tests.py` in the `documentation/deliberately_vulnerable_flask_app` and importing the helper functions from `tools/integration_tests/integration_test_utils.py `
2. Modify `run_integration_tests.py` to execute `run_pysa_analysis ` with an extra passthrough argument "-l" pointing to the `documentation/deliberately_vulnerable_flask_app` where the pyre configuration is located

Reply:
Will go with the first approach as it looks much more cleaner :)

Reply:
Hey got another option, will go with this:
If we run integration tests script from any directory previously it tried to run Pysa on the directory the script was at.
Instead, if we modify the script in such a way that it runs Pysa on the current working directory, it should work and there would be no need to refractor, move files, or even add extra arguments.

I feel this is cleaner, hope it works, can change if you want tho!